### PR TITLE
fix: update `php-parser`

### PR DESCRIPTION
**Pull request by evilebottnawi**
Update parser:
- comment for constants in right places how

### CODE REVIEW

File: tests/comments/__snapshots__/jsfmt.spec.js.snap [hunk 1]
```diff
@@ -987,8 +987,7 @@ $var = new class {
 };
 
 $var = new class {
-    const CONSTANT =
-        // Comment 1
+    const CONSTANT = // Comment 1
```
Comment (evilebottnawi) [inline tests/comments/__snapshots__/jsfmt.spec.js.snap]: As original input

**Comment by evilebottnawi** [comment]
Travis has problem with invalid node_modules and using branch hashes :confused:

**Comment by czosel** [comment]
Hmm, did you already try clearing the travis cache?

**Comment by evilebottnawi** [comment]
@czosel yep, it is green only when caches were cleaned

**Comment by evilebottnawi** [comment]
I will write in bug tracker travis in near future

**Pull request merged**